### PR TITLE
Use github code owners instead of pullappprove

### PR DESCRIPTION
**Pull request by goetas**
pullapprove is a paid service now, github code owners offers a similar feature

### CODE REVIEW

**Comment by ashnazg** [comment]
The decision to change LGTM

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/proposed/ @php-fig/secretaries
+/bylaws/ @php-fig/secretaries
+/proposed/security* @michaelcullum @php-fig/psr-9-10
```
Comment (Jean85) [inline CODEOWNERS]: Michael is included in that team, so it shouldn't be needed here

```suggestion
/proposed/security* @php-fig/psr-9-10
```

**Comment by michaelcullum** [comment]
Should also probably have a catch-all `*` for secretaries. Shame code owners doesn't work for labels. That was nice.

Ah well.

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/proposed/ @php-fig/secretaries
```
Comment (michaelcullum) [inline CODEOWNERS]:
```suggestion
* @php-fig/secretaries
/proposed/ @php-fig/secretaries
```

**Comment by goetas** [comment]
suggested changes applied

**Comment by goetas** [comment]
Then it is!

**Pull request merged**